config.js

```diff
@@ -0,0 +1,75 @@
+// NOTE: Neither of these files are committed and for good reason.
```

Comment:
I agree entirely regarding segregation of the keys.

The concern was whether or not the config could be replaced easily.

I dont want to have to push changes to chesster when the league format changes - unless it changes in a way that requires changing the format of the config - that is a different story - but this format of config seems flexible enough.